solutions/observability/get-started/what-is-elastic-observability.md

```diff
@@ -19,11 +19,11 @@ mapped_pages:
 
 Analyze log data from your hosts, services, Kubernetes, Apache, and many more.
 
-In **Logs Explorer** (powered by Discover), you can quickly search and filter your log data, get information about the structure of the fields, and display your findings in a visualization.
+In **Discover**, you can quickly search and filter your log data, get information about the structure of the fields, and display your findings in a visualization.
 
-:::{image} /solutions/images/serverless-log-explorer-overview.png
-:alt: Logs Explorer showing log events
-:screenshot:
+:::{image} ../../images/serverless-log-explorer-overview.png
```

Comment:
This img is from Logs Explorer and although it's very similar to Discover it has some differences (namespace control and the name of the "All log datasets"

<img width="867" alt="image" src="https://github.com/user-attachments/assets/6739d688-a53a-450c-a72a-2fcd89e9eed1" />